src/shapes/ActiveSelection.ts

```diff
@@ -47,8 +47,11 @@ export class ActiveSelection extends Group {
    * Adds objects with respect to {@link multiSelectionStacking}
    * @param targets object to add to selection
    */
-  multiSelectAdd(...targets: FabricObject[]) {
-    if (this.multiSelectionStacking === 'selection-order') {
+  addToSelection(mode: 'multi-select' | 'default', ...targets: FabricObject[]) {
+    if (
+      mode === 'default' ||
```

Comment:
`default` isn't really a good name. What's default? It doesn't seem to do much, the only real difference in the implementation is `multiSelectionStacking`. So the use cases:

```ts
activeSelection.addToSelection('multi-select', activeObject, target);
this._activeSelection.addToSelection('default', ...objects);
```

can be changed to:

```ts
activeSelection.addToSelection(activeObject, target, { multiSelectionStacking: true });
this._activeSelection.addToSelection(...objects,  { multiSelectionStacking: false }); // equivalent to mode default
```

By adding a `mode` you're making the method much harder to reason about as the technically combinations have cardinality `mode x multiSelectionStacking`, 2x2=4 in this case. At the end of the day, the difference is in how objects are added to the selection, not the "mode" right?